src/IIIFPresentation/API/Features/Manifest/CanvasPaintingMerger.cs

```diff
@@ -92,7 +93,8 @@ private void JoinPaintedResourcesWithItems(List<InterimCanvasPainting> itemsCanv
         {
             var matchedPaintedResourceCanvasPaintings = paintedResourceCanvasPaintings.Where(cp =>
                 !string.IsNullOrEmpty(cp.Id) && string.Equals(cp.Id, itemsCanvasPainting.Id,
-                    StringComparison.OrdinalIgnoreCase)).ToList();
+                    StringComparison.OrdinalIgnoreCase)).OrderBy(cp => cp.CanvasOrder)
```

Comment:
I think `.OrderBy(cp => cp.CanvasOrder).ThenBy(cp => cp.ChoiceOrder)` is used in a few places - is it worth having this as a reusable method?